build.gradle.kts

```diff
@@ -114,29 +117,66 @@ subprojects {
             html.enabled(true)
             xml.enabled(true)
         }
-
-        tasks.withType<Javadoc>().all { enabled = false }
     }
 
     if (!isSample) {
         apply {
-            plugin(BintrayRelease.plugin)
-            plugin(Ktlint.plugin)
-        }
+            plugin("org.gradle.maven-publish")
 
-        configure<PublishExtension> {
-            artifactId = project.name
-            autoPublish = true
-            desc = "The easiest HTTP networking library in Kotlin/Android"
-            groupId = "com.github.kittinunf.fuel"
-            setLicences("MIT")
-            publishVersion = Fuel.publishVersion
-            uploadName = "Fuel-Android"
-            website = "https://github.com/kittinunf/Fuel"
+            plugin(Release.Bintray.plugin)
+            plugin(Ktlint.plugin)
         }
 
         configure<KotlinterExtension> {
             reporters = arrayOf(ReporterType.plain.name, ReporterType.checkstyle.name)
         }
+
+        version = Fuel.publishVersion
+        configure<BintrayExtension> {
+            user = findProperty("BINTRAY_USER") as? String
+            key = findProperty("BINTRAY_KEY") as? String
+            dryRun = true
+            setConfigurations("archives")
+            pkg(delegateClosureOf<BintrayExtension.PackageConfig> {
+                repo = "maven"
+                name = "Fuel-Android"
+                desc = "The easiest HTTP networking library in Kotlin/Android"
+                userOrg = "kittinunf"
+                websiteUrl = "https://github.com/kittinunf/Fuel"
+                vcsUrl = "https://github.com/kittinunf/Fuel"
+                setLicenses("MIT")
+                version(delegateClosureOf<BintrayExtension.VersionConfig> {
+                    name = Fuel.publishVersion
+                })
+            })
+        }
+
+        if (project.hasProperty("android").not()) {
+            val sourcesJar by tasks.registering(Jar::class) {
+                classifier = "sources"
+                from(sourceSets["main"].allSource)
+            }
+
+            val doc by tasks.registering(Javadoc::class) {
```

Comment:
this can be changed to `creating` and it doesn ot have to be accessed with `.get()`

Reply:
I think I need to use the `.get()` still after changed to `creating` ...